src/config.ts

```diff
@@ -40,6 +40,12 @@ const dynamoDbBundleService = new DynamoDbBundleService(DynamoDb, undefined, und
     enableMultiTenancy,
 });
 
+// Lambda payload limit is 6MB, assuming an average request of 4KB,
```

Comment:
Let's put this in documentation instead of in config file. If the const `maxBatchSize` is not used, it should not be defined.

Reply:
Added documentation and removed the comment+variable declaration from config.ts